tests/test_obstacle_map.py

```diff
@@ -0,0 +1,107 @@
+from SDA import *
+import unittest
+import numpy 
+
+class Test_Obstacle_map(unittest.TestCase):
+	"""docstring for test_Obstacle_map"""
```

Comment:
Please remove the docstring. All you are saying is that it is a docstring for obstacle map...